plugins/chat/spec/system/thread_tracking/full_page_spec.rb

```diff
@@ -62,5 +62,37 @@
       channel_page.open_thread_list
       expect(thread_list_page).to have_no_unread_item(thread.id)
     end
+
+    it "allows the user to change their tracking level for an existing thread" do
+      chat_page.visit_thread(thread)
+      thread_page.change_notification_level(:normal)
+      expect(thread.reload.membership_for(current_user).notification_level).to eq("normal")
+    end
+
+    it "allows the user to start tracking a thread they have not replied to" do
+      new_thread = Fabricate(:chat_thread, channel: channel)
+      Fabricate(:chat_message, chat_channel: channel, thread: new_thread)
+      chat_page.visit_thread(new_thread)
+      thread_page.change_notification_level(:tracking)
+      expect(new_thread.reload.membership_for(current_user).notification_level).to eq("tracking")
+      chat_page.visit_channel(channel)
+      channel_page.open_thread_list
+      expect(thread_list_page).to have_thread(new_thread.id)
```

Comment:
if you want to use id, it's probably better to have an API like `have_thread(id: ...)`